src/components/Banner.astro

```diff
@@ -1,12 +1,21 @@
 ---
-function t(input) {
-  return input;
-}
+import { getLocale } from "astro-i18n-aut";
+
+// TODO: How to handle HTML?
 // import { Trans } from "astro-i18next/components";
+
 // assets
 import FLAG_PNG from "@uswds/uswds/img/us_flag_small.png";
 import GOV_SVG from "@uswds/uswds/img/icon-dot-gov.svg";
 import HTTPS_SVG from "@uswds/uswds/img/icon-https.svg";
+// utils
+import { t as baseT } from "@utils/t.js";
+
+// If locale is undefined, then it uses defaultLocale
+const locale = getLocale(Astro.url);
+function t(key) {
+  return baseT(key, locale);
+}
```

Comment:
Neat! This could be a way to clean this up, if the whole `baseT` thing is something you need to do in every file.

```js
// @utils/getTrans.js (or some other util)
import { getLocale } from 'astro-i18n-aut'
import { t } from '@utils/t.js'

export const getTrans = (url) => {
  const locale = getLocale(url)
  return (key) => t(key, locale)
}

// src/components/Banner.astro
import { getTrans } from '@utils/getTrans.js'
const t = getTrans(Astro.url)
```